### ISSUE TITLE
[Bug]: Screenshots with a shader using post-processing tonemapping are incredibly washed out and overexposed

### ISSUE THREAD

**Issue by steve-the-player** [Minenash/Seamless-Loading-Screen#101]
![2024-03-02_00 25 53](https://github.com/Minenash/Seamless-Loading-Screen/assets/61434698/ce8e52b2-c171-4837-9d34-5d6312f5dac7)
<img width="1512" alt="Screenshot 2024-03-02 at 12 26 03 AM" src="https://github.com/Minenash/Seamless-Loading-Screen/assets/61434698/54b856df-65b3-4c9a-9495-028aada16ac2">

~~This appears to happen more often with night vision for some reason.~~ _Edit: not actually true, it was just the simulated autoexposure being weird when night vision was enabled. It's just as bad without night vision_  
I'm using Complementary shaders in this screenshot, but I've observed the same behavior from BSL when tonemapping was enabled

**Comment by k2helix** [comment]
This also happens with latest Bliss Shaders. Disabling the Auto-Exposure option and setting it manually helps but the screenshot still feels overexposed.

**Comment by Avamander** [comment]
Yeah, I have the same issue with BSL shaders. Fabrishot fixed this by averaging multiple frames for the screenshot (though vanilla/F2 screenshots do not exhibit this issue).
